projectread/settings.py

```diff
@@ -32,6 +32,7 @@
 
 ALLOWED_HOSTS = [] if DEBUG else ["project-read-backend.herokuapp.com"]
 
+CORS_ALLOWED_ORIGINS = ["http://localhost:3000"] if DEBUG else []
```

Comment:
would the else clause just have the origin for our front-end in the future?

Reply:
yep that's right! we'll have to add the url when we deploy it